extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -622,6 +642,28 @@ protected function actionEvaluate() {
 		}
 	}
 
+	protected function actionTestDiscussionsActivity() {
+		global $wgCityId;
+		$ids = $this->ids;
+		if ( empty($ids) ) {
+			$ids = array( $wgCityId );
+		}
+		echo "Checking discussions activity:\n";
+
+		foreach ($ids as $id) {
+			$isActiveSite = true;
+
+			try {
+				$isActiveSite = $this->isActiveSite((int)$id);
```

Comment:
There's no need to cast this to an `int`; PHP will cast automatically as necessary.  I know whoever wrote this code felt compelled to use `intval()` liberally, but it just adds noise.